contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol

```diff
@@ -181,21 +182,23 @@ contract FunctionsBillingRegistry is
     uint256 gasAfterPaymentCalculation,
     int256 fallbackWeiPerUnitLink,
     uint32 gasOverhead,
-    uint32 requestTimeoutSeconds
+    uint32 requestTimeoutSeconds,
+    uint96 fee
   ) external onlyOwner {
     if (fallbackWeiPerUnitLink <= 0) {
       revert InvalidLinkWeiPrice(fallbackWeiPerUnitLink);
     }
-    s_config = Config({
```

Comment:
Maybe rename "s_config" to "s_config_DEPRECATED" to make it clear? Unless I'm misunderstanding the post-upgrade step.

Reply:
We can't rename of the old piece of state